src/gui/mainwindow.cpp

```diff
@@ -1095,15 +1104,31 @@ void MainWindow::dropEvent(QDropEvent *event)
         files = event->mimeData()->text().split('\n');
     }
 
-    // Add file to download list
+    // differentiate ".torrent" files and others
+    QStringList torrentFiles, otherFiles;
```

Comment:
Do not insist, but I would use `std::partition()` here without explicit loop and two additional containers.